### PR TITLE
QUIZ-133 Setup GraphQL codegen

### PR DESCRIPTION
**Pull request by BenLanglois**
In this PR:

- Setup TypeScript code generation from the real GraphQL schema from the backend
- Updated the readme with instructions on how to run the codegen
- Updated code to match the properties of questions/answers from the schema. Notably, the question type no longer has a `type` field, and the `question` field was renamed to `text`.

As discussed on discord, I'll work on the types for queries/migrations in another PR.

### CODE REVIEW

**Comment by heathermusson** [comment]
@ariellam or @alexgirard can one of you review this? i am blocked by this PR before i can merge one of mine.

File: src/instructor/CreateQuizPage.tsx [hunk 1]
```diff
@@ -30,9 +30,9 @@ interface Props {
 const dummyQuestions: Question[] = [
   {
     id: uuid(),
-    question: 'Question Text',
-    type: 'Multiple Choice',
+    text: 'Question Text',
     format: 'MC',
+    hasLabels: false,
```
Comment (alexgirard) [inline src/instructor/CreateQuizPage.tsx:35]: why was this prop added? i thought it was optional

**Review by alexgirard — APPROVED** [review]
lgtm (discussed in weekly meeting)

File: src/instructor/CreateQuizPage.tsx (hunk 1 above)
Comment (BenLanglois) [reply src/instructor/CreateQuizPage.tsx:35]: It's not optional on the GraphQL `Question` type, which the new `Question` type is based on. Removing `hasLabels: false` causes the compiler to throw an error.

**Pull request merged**